runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/control/DefaultJobBundleFactory.java

```diff
@@ -374,17 +374,21 @@ ServerInfo getServerInfo() {
       return serverInfo;
     }
 
-    @Override
-    public void close() throws Exception {
+    public void close() {
```

Comment:
```suggestion
    public void close() {
    Exception closeException = null;
```

Reply:
How about constructing an exception `closeException` here to collect the exceptions?